CHANGES.md

```diff
@@ -1,5 +1,10 @@
 # Change Log
 
+# 1.1.4
+
+- Point requirements.txt to a fork of flanker with the tld package pinned to 0.10. See https
+://github.com/StackStorm-Exchange/stackstorm-email/issues/27 for more info.
```

Comment:
You can just reference `#27` instead of using full URL, - a little bit nicer.

Also making a small note in README _why_ this fix is there (eg. to keep py2 support) would be nice.